src/platform-implementation-js/dom-driver/gmail/views/gmail-app-sidebar-view/primary/index.tsx

```diff
@@ -374,9 +378,19 @@ class GmailAppSidebarPrimary {
         buttonContainer = document.createElement('div');
         buttonContainer.className = idMap('sidebar_button_container');
         buttonContainer.setAttribute('data-app-name', appName);
-        buttonContainer.innerHTML = autoHtml`
-          <button class="inboxsdk__button_icon ${iconClass}" type="button" data-tooltip="${appName}">
-            <img class="inboxsdk__button_iconImg" src="${iconUrl}">
+        buttonContainer.innerHTML = `
+          <button class="inboxsdk__button_icon ${escape(
+            iconClass,
+          )}" type="button" data-tooltip="${escape(appName)}">
```

Comment:
would changing it to escaping these cause any issues for existing usages?

Reply:
I don't think so. `auto-html` doesn't support conditionals, but it does use `_.escape` under the covers for values. This is just falling back to that level.